apel/db/loader/record_factory.py

```diff
@@ -115,6 +149,43 @@ def create_records(self, msg_text):
     # Private methods below
     ######################################################################
 
+    def _validate_json(self, js, schema):
+        '''Run jsonschema validate on JSON given a schema otherwise raise a friendly error'''
```

Comment:
Should we need to convert single quotes to double quotes across code

Reply:
Double quotes would be better.